.circleci/config.yml

```diff
@@ -18,39 +18,43 @@ jobs:
       - checkout: *post_checkout
       - restore_cache: *_root_package_lock_key
       - run: yarn install --frozen-lockfile
```

Comment:
This install has a couple of warnings on CI:
```
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @angular-devkit/build-ng-packagr@0.0.0" has unmet peer dependency "ng-packagr@^2.2.0 || ^3.0.0 || ^4.0.0".
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @angular-devkit/build-webpack@0.0.0" has unmet peer dependency "webpack@^4.6.0".
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @angular-devkit/build-webpack@0.0.0" has unmet peer dependency "webpack-dev-server@^3.1.4".
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @ngtools/webpack@0.0.0" has unmet peer dependency "webpack@^4.0.0".
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @angular-devkit/build-ng-packagr > ng-packagr@4.0.1" has unmet peer dependency "@angular/compiler@^6.0.0".
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @angular-devkit/build-ng-packagr > ng-packagr@4.0.1" has unmet peer dependency "@angular/compiler-cli@^6.0.0".
warning "workspace-aggregator-15c1ff0e-53e5-4313-986b-128baf16c8df > @angular-devkit/build-ng-packagr > ng-packagr@4.0.1" has unmet peer dependency "tslib@^1.9.0".
```

Reply:
Yes.  This is a yarn issue.  It is not accounting for hoisted dev dependencies in workspaces.
ref: https://github.com/yarnpkg/yarn/issues/5810